docs/introduction/build_windows.md

```diff
@@ -1,84 +1,99 @@
-To build Thunder and its components on Windows, you will need Visual Studio installed -  the free community edition is just fine.
+To build the Thunder framework and its components on Windows, you need to have Visual Studio installed. The free Community Edition is sufficient.
```

Comment:
I see that it is not new here but should we perhaps mention something like "you need to be check if you are entitled to use the free version in your situation, please check the license" before people will say we encouraged them to do so :)
(just checked and sometimes it is even allowed in a commercial situation, e.g. for relatively small companies for developing open source software)

Reply:
Good idea, better safe than sorry, I made the change:
_"The free Community Edition is sufficient if you are entitled to use it in your situation, and to make sure of that please check the license."_